worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -1712,7 +1803,7 @@ public int stackCuboidRegion(Region region, BlockVector3 dir, int count, boolean
      * Stack a cuboid region.
      *
      * @param region       the region to stack
-     * @param dir          the direction to stack
+     * @param offset how far to move the contents each stack
```

Comment:
It's not just an offset, just as much as it's not just a direction. The variable should reflect that it's a direction accepting a "distance" so if the effective vector length is greater than 1, then there is an "offset".